src/scripts/self-hosted-runners/setup-self-hosted-runner-ubuntu.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
The name of this scrip should be changed to setup-rootless-docker; hence we don't actually setup the runner here, only installing rootless docker, same to the centos one

Reply:
done